src/constructs/cloudwatch/ec2-alarms.ts

```diff
@@ -62,23 +62,27 @@ export class Gu5xxPercentageAlarm extends GuAlarm {
 export class GuUnhealthyInstancesAlarm extends GuAlarm {
   constructor(scope: GuStack, props: GuUnhealthyInstancesAlarmProps) {
     const alarmName = `Unhealthy instances for ${props.app} in ${scope.stage}`;
-    const alarmDescription = `${props.app}'s instances have failed healthchecks several times over the last hour.
+
+    const period = Duration.minutes(5);
+    const evaluationPeriods = 12;
+    const evaluationInterval = Duration.minutes(period.toMinutes() * evaluationPeriods).toHumanString();
```

Comment:
`period.toMinutes()` assumes `period` will always be a minute `Duration`, `periodInMinutes` might be a more explicit variable name?